src/node.cc

```diff
@@ -2261,6 +2261,41 @@ static void Uptime(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void GetResourceUsage(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  uv_rusage_t rusage;
+  int err = uv_getrusage(&rusage);
+  if (err)
+    return env->ThrowUVException(err, "uv_getrusage");
+
+  Local<Function> fn = args[0].As<Function>();
+  Local<Value> argv[] = {
+    Number::New(env->isolate(), rusage.ru_utime.tv_sec),
```

Comment:
Minor nit, can you capture `Isolate* isolate = env->isolate()` above and just make these `Number::New(isolate, ...` to make the code slightly more concise.